src/Behat/Behat/Gherkin/Specification/Locator/FilesystemFeatureLocator.php

```diff
@@ -132,11 +132,17 @@ private function findFeatureFiles($path)
         }
 
         $iterator = new RegexIterator(
-            new RecursiveIteratorIterator(
-                new RecursiveDirectoryIterator($absolutePath)
-            ), '/^.+\.feature$/i',
+            new recursiveIteratorIterator(
+                new RecursiveDirectoryIterator(
+                    $absolutePath,
+                    RecursiveDirectoryIterator::FOLLOW_SYMLINKS |
+                    RecursiveDirectoryIterator::SKIP_DOTS
```

Comment:
this should be on the previous line to be more readable. It is the same argument, not a different argument